pandas/core/groupby.py

```diff
@@ -191,6 +191,107 @@
     dtype: int64
     """)
 
+_numeric_operations_full_names = dict(
+    sum='the sum',
+    prod='the product',
+    min='the minimum',
+    max='the maximum',
+    first='the first value',
+    last='the last value'
+)
+
+_numeric_operations_doc_template = dedent(
+    """
+    Compute %(full_name)s of each group.
+
+    For multiple groupers, the result index will be a
+    :class:`~pandas.MultiIndex`.
+
+    Parameters
+    ----------
+    kwargs : dict
+        Optional keyword arguments to pass to `%(f)s`.
+
+        `numeric_only` : bool
+            Include only float, int, boolean columns.
+        `min_count` : int
+            The required number of valid values to perform the operation.
+            If fewer than `min_count` non-NA values are present the result
+            will be NA.
+
+    Returns
+    -------
+    Series or DataFrame
+    {examples}
+    {see_also}
+    """
+)
+
+_numeric_operations_examples = dict()
+
+_numeric_operations_examples['max'] = dedent(
+    """
+    Examples
+    --------
+    >>> df = pd.DataFrame(
+    ...     {'type': ['apple', 'apple', 'apple', 'orange', 'orange'],
+    ...      'variety': ['gala', 'fuji', 'fuji', 'valencia', 'navel'],
+    ...      'quantity': [2, 4, 8, 3, 1],
+    ...      'price': [0.8, 1.25, 2.5, 1.25, 1.0],
+    ...     },
+    ...     columns=['type', 'variety', 'quantity', 'price']
+    ... )
+    >>> df
+         type   variety  quantity  price
+    0   apple      gala         2   0.80
+    1   apple      fuji         4   1.25
+    2   apple      fuji         8   2.50
+    3  orange  valencia         3   1.25
+    4  orange     navel         1   1.00
+
+    >>> g = df.groupby('type')
+    >>> g.max()
+             variety  quantity  price
+    type
+    apple       gala         8   2.50
+    orange  valencia         3   1.25
+
+    By default, the ``max`` operation will be applied to all columns (including
+    the 'variety' column which is of type `str`).
+
+    Specifying ``numeric_only=True`` will keep only the numeric columns
+    ('quantity' and 'price' in this example).
+
+    >>> g.max(numeric_only=True)
+            quantity  price
+    type
+    apple          8   2.50
+    orange         3   1.25
+
+    Grouping by more than one column results in :class:`~pandas.DataFrame` with
+    a :class:`~pandas.MultiIndex`.
+
+    >>> g = df.groupby(['type', 'variety'])
+    >>> g.max()
+                     quantity  price
+    type   variety
+    apple  fuji             8   2.50
+           gala             2   0.80
+    orange navel            1   1.00
+           valencia         3   1.25
+    """  # noqa
+)
+
+_numeric_operations_see_also = dedent(
+    """See Also
```

Comment:
You need to have the "See also" on the next line, otherwise `dedent` does not work. 
But if this gives a blank line too much, I think you can do:

```
    """\
    See Also
    ...
```